vic5_manuscript.tex

```diff
@@ -159,7 +159,7 @@ \section{VIC Infrastructure}
 
-  While VIC's original infrastructure may have made coupling within ESMs difficult, it also contributed to the model's wide uptake throughout the hydrologic modeling community. Previous versions of VIC were very portable, required no third-party libraries, were easy to run in single point mode for testing or comparison with point observations, and were trivial to parallelize across many processors and machines by manually subdividing the domain and running each sub-domain on a separate processor. It was this simple infrastructure, combined with the inclusion of built-in convenience tools such as the meteorological variable disaggregator based on the MT-CLIM algorithms \citep{Thornton_1999,Bohn_2013}, publicly available regional and global forcing and parameter datasets, and publicly available source code, that greatly contributed to VIC's wide adoption in the hydrologic modeling community.
+  While VIC's original infrastructure may have made coupling within ESMs difficult, it also contributed to the model's wide uptake throughout the hydrologic modeling community. Previous versions of VIC were very portable, required no third-party libraries, were easy to run in single point mode for testing or comparison with point observations, and were trivial to parallelize across many processors and machines by manually subdividing the domain and running each sub-domain on a separate processor. It was this simple infrastructure, combined with the inclusion of built-in convenience tools such as the meteorological variable disaggregator based on the Mountain Climate Simulator (MT-CLIM) algorithms \citep{Thornton_1999,Bohn_2013}, publicly available regional and global forcing and parameter datasets, and publicly available source code, that greatly contributed to VIC's wide adoption in the hydrologic modeling community.

```

Comment:
Just to be annoyingly technical, might be worth additionally citing Kimball et al 1997 (or 1998?) for the VP algorithm, for completeness.

Reply:
Nope - people can look at Bohn 2013 or any of the others to get the detailed info. That is not what this paper is about.

Reply:
OK, but same can be said for Thornton and Running 1999, which is cited.  Perhaps Thornton and Running should be omitted.